packages/clerk-js/src/ui/elements/__tests__/PlainInput.test.tsx

```diff
@@ -120,7 +120,7 @@ describe('PlainInput', () => {
     expect(getByRole('link', { name: /take action/i })).toBeInTheDocument();
     expect(getByRole('link', { name: /take action/i })).not.toHaveAttribute('rel');
     expect(getByRole('link', { name: /take action/i })).not.toHaveAttribute('target');
-    expect(getByRole('link', { name: /take action/i })).toHaveAttribute('href', '');
+    expect(getByRole('link', { name: /take action/i })).toHaveAttribute('href', '#');
```

Comment:
Seems like the upgraded package will not match the `link` role to an anchor tag with an empty href attribute.
A quick test verified that if we use `#` as the href value, the tests will pass and the intended usage of `a` will not change within our component.

Feels like this anchor tag needs to be replaced with a button, but i can see how this could be considered a breaking change.

Reply:
Alternatively we can change the tests to something like
```tsx
expect(getByText(/take action/i, { selector: 'a' })).toBeInTheDocument();
```

Reply:
`href="#"` typically scrolls the page to the top, that could be an unintended side-effect of this change.

I also noticed while working on the E2E tests for Elements that this shouldn't be a `<link>` but a `<button>`. Can you add a `TODO(v6)` to make this change?

Reply:
Yeah, i agree that this should be a button.

Reply:
I think i will proceed with just updating the tests then.